RecoTracker/SiTrackerMRHTools/plugins/SiTrackerMultiRecHitUpdatorESProducer.cc

```diff
@@ -1,41 +1,60 @@
-#include "RecoTracker/SiTrackerMRHTools/plugins/SiTrackerMultiRecHitUpdatorESProducer.h"
-#include "TrackingTools/Records/interface/TransientRecHitRecord.h"
-#include "RecoTracker/Record/interface/CkfComponentsRecord.h"
-
-#include "FWCore/Framework/interface/EventSetup.h"
+// user includes
 #include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
Not needed any more...

Reply:
removed in [0a9ee2d](https://github.com/cms-sw/cmssw/pull/36980/commits/0a9ee2def74a0cf443e5e22a5b52be489d5a03de)